### DISCUSSION TITLE
码表“码”列的分隔符是什么

### DISCUSSION THREAD

**Discussion by Ynjxsjmh** [rime/home#1015] [Q&A] [answered]
[碼表與詞典 | Rime 輸入方案設計書 · rime/home Wiki](https://github.com/rime/home/wiki/RimeWithSchemata#%E7%A2%BC%E8%A1%A8%E8%88%87%E8%A9%9E%E5%85%B8) 这一小节没有介绍詞典文件的前半部的 YAML 文檔可以有 columns 字段，这个字段是在 [描述檔 | Schema.yaml 詳解 · LEOYoon-Tsaw/Rime_collections/Rime_description.md](https://github.com/LEOYoon-Tsaw/Rime_collections/blob/master/Rime_description.md#%E6%8F%8F%E8%BF%B0%E6%AA%94-1) 这里介绍的，里面说

> columns: 定義碼表以Tab分隔出的各列，可設text【文本】、code【碼】、weight【權重】、stem【造詞碼】

我看到有的配置写的是下面这样的：

```
# emoji.dict.yaml
# 雪齋 <leoyoontsaw@gmail.com>
# 編碼以 Unicode 分類爲主，Kunki Chou 整理
# 29-Apr-2014 添加字頻，字頻得自 http://www.emojitracker.com ，統計自 twitter.com

---
name: emoji
version: "2.5"
sort: by_weight
use_preset_vocabulary: false
columns:
  - text
  - code
  - weight
...
#天氣
☀	sunny;tian'qi`weather	10908516
☔	rain;tian'qi`weather	951774
⛈	storm;tian'qi`weather
⛆	haze;tian'qi`weather
```

这个 `code【碼】` 列用 `;` 和 ``` 分隔成了三个字段，用这三个字段的任何一个字段都可以搜索，请问这个分隔符有什么讲究么？

**Comment by lotem** [comment, marked as answer]
沒有。那些符號是編碼的一部分。
